app/controllers/books_controller.rb

```diff
@@ -21,10 +21,9 @@ def index
   def show
     book = Book.find_by(id: params[:id])
 
-    if book.nil?
-      redirect_to books_path
-    else
-      @book = book.decorate
-    end
+    redirect_to books_path if book.nil?
+
+    @book = book.decorate
+    @reviews = Review.where(book_id: @book.id, verified: true)
```

Comment:
```suggestion
    @reviews = @book.reviews.verified.order(created_at: desc)
```
P.S. I'm not sure about ordering, check the specs